### PR TITLE
Add `iobroker extend object` command

### PR DESCRIPTION
**Pull request by AlCalzone**
Fixes: #271

### CODE REVIEW

**Comment by AlCalzone** [comment]
@Apollon77 Appveyor seems to have random build failures again :(

**Comment by Apollon77** [comment]
I know :-(

File: lib/cli/cliObjects.js [hunk 1]
```diff
@@ -227,6 +229,36 @@ module.exports = class CLIObjects extends CLICommand {
...
+        const parsedArg = parsePropPathAndAssignment(lastArg);
+        // extend does not accept a property path, so error when one is passed
+        if (!parsedArg || parsedArg.propPath) {
+            CLI.error.invalidJSONValue();
+            return void callback(3);
+        }
+        const { value } = parsedArg;
+
+        dbConnect(objects => {
+            objects.extendObject(id, value, () => {
```
Comment (Apollon77) [inline lib/cli/cliObjects.js:255]: No error checking? Always success? :-)

**Comment by Apollon77** [comment]
Thank you!

**Pull request merged**